src/plugins/cordova-plugin-globalization/sim-host-panels.html

```diff
@@ -1,7 +1,22 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved. -->
 
 <cordova-panel id="globalization" caption="Globalization" spoken-text="Globalization Panel" data-loc-id="5ffc4b1e">
-    <cordova-combo id="locale-list" label="Locale" data-loc-id="9e831385"></cordova-combo>
+    <cordova-combo id="locale-list" label="Locale" data-loc-id="9e831385">
+    	<option value="en-US" data-loc-id="0ebb8953">English</option>
```

Comment:
Generally I'd leave out the `data-loc-id` attributes until the commit with the `update-loc` run, but including them now should be fine.